tests/dune

```diff
@@ -8,6 +8,16 @@
     console.lib
     pastel-console.lib
   )
+  (modules (:standard \ ReasonNativeTests))
```

Comment:
This is the only part that's a little weird here.

Reply:
I agree, I think that it is ultimately worth it given that this is a fixed cost and the aggregate result of this changes makes the relationship between the tests library and executable more apparent.